Jenkinsfile

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/groovy
+@Library('github.com/stakater/fabric8-pipeline-library@master')
+
+def utils = new io.fabric8.Utils()
```

Comment:
@waseem-h why is this in outer block whereas everything else is defined inside the container block?